pkg/util/globalerror/errors.go

```diff
@@ -68,9 +68,21 @@ func (id ID) Message(msg string) string {
 	return fmt.Sprintf("%s (%s%s)", msg, errPrefix, id)
 }
 
-// MessageWithLimitConfig returns the provided msg, appending the error id and a suggestion on
-// which configuration flag(s) to use to change the limit.
-func (id ID) MessageWithLimitConfig(msg, flag string, addFlags ...string) string {
+// MessageWithPerInstanceLimitConfig returns the provided msg, appending the error id and a suggestion on
+// which configuration flag(s) to use to change the per-instance limit.
+func (id ID) MessageWithPerInstanceLimitConfig(msg, flag string, addFlags ...string) string {
```

Comment:
I think optimally we would avoid using the term "instance", because previously we used to consider a "tenant" and various other things to be "instances" as well. How about "process" instead of "instance"?

Reply:
We can open an issue about it. The code currently reflects the user-facing naming (such config is called per-instance both in the config and doc).